openai/src/commonMain/kotlin/com/xebia/functional/xef/conversation/llm/openai/OpenAI.kt

```diff
@@ -138,6 +132,28 @@ class OpenAI(internal var token: String? = null, internal var host: String? = nu
       DALLE_2,
     )
 
+  suspend fun findModel(modelId: String): Any? { // TODO: impl of abstract provider function
```

Comment:
This method should be private and return `ModelType?` not `Any?`

```suggestion
  private suspend fun findModel( modelId: String): ModelType?
```